fourth-wall.js

```diff
@@ -52,27 +53,32 @@
     };
 
     FourthWall.parseGistData = function (gistData, that) {
-        var objects = [];
+        var config = [];
         for (var file in gistData.data.files) {
             if (gistData.data.files.hasOwnProperty(file)) {
                 var filedata = gistData.data.files[file],
                     lang = filedata.language;
 
-                if (lang == 'JavaScript' || lang == 'JSON' || lang == null) {
-                    var o = JSON.parse(filedata.content);
-                    if (o) {
-                        objects.push(o);
+                if (file == 'users.json') {
+                    var users_file = filedata.content
```

Comment:
JavaScript style is to use camelCase rather than underscores. So this should be `usersFile` and the config file variable below should be `configFile`. This matches the rest of the file.